src/test/scala/ch.zzeekk.spark.temporalquery/TestUtils.scala

```diff
@@ -35,7 +35,7 @@ object TestUtils extends Logging {
   def printFailedTestResult(testName: String, arguments: Seq[DataFrame])(actual: DataFrame)(expected: DataFrame): Unit = {
     def printDf(df: DataFrame): Unit = {
       println(df.schema.simpleString)
-      df.orderBy(df.columns.head,df.columns.tail:_*).show(false)
+      df.show(false)
```

Comment:
One cannot simply order by all columns if a column name occurs twice. Unfortunately df.columns does not contain the data frame alias from the origin.